pkg/kubelet/dockertools/manager.go

```diff
@@ -1919,7 +1919,7 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, runningPod kubecontainer.Pod, pod
 			continue
 		}
 
-		if container.SecurityContext != nil && container.SecurityContext.RunAsNonRoot {
+		if container.SecurityContext != nil && container.SecurityContext.RunAsNonRoot != nil && *container.SecurityContext.RunAsNonRoot {
```

Comment:
I had the same thought, but my usual threshold is to introduce those when I start to have to repeat myself.  You okay with this for now, and I will introduce a helper (where I think I will need it to avoid repeating myself) in #14705 ?